docs/docs/node-apis.md

```diff
@@ -1,15 +1,15 @@
 ---
 title: Gatsby Node APIs
-description: Documentation on Node APIs used in Gatsby build process for common uses like creating pages
+description: पृष्ठे तयार करण्यासारख्या सामान्य वापरासाठी Gatsby बिल्ड प्रक्रियेत वापरल्या जाणार्‍या नोड एपीआयवरील दस्तऐवजीकरण
```

Comment:
दस्तऐवजीकरण => it should be documentation as it is

Reply:
also dont translate pages to पृष्ठे, keep as is